osm2gtfs/creators/trips_creator.py

```diff
@@ -13,32 +19,301 @@ def __repr__(self):
+        """
+        Generate a transitfeed ServicePeriod object
+        from a time string according to the standard schedule:
+        https://github.com/grote/osm2gtfs/wiki/Schedule
+
+        :return: ServicePeriod object
+        """
+        try:
+            gtfs_service = feed.GetServicePeriod(service)
+            if gtfs_service is not None:
+                return gtfs_service
+        except KeyError:
+            pass
+
+        if service == "Mo-Fr":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(False)
+        elif service == "Mo-Sa":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(5, True)
+        elif service == "Mo-Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(True)
+        elif service == "Sa":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(5, True)
+        elif service == "Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(6, True)
+        elif service == "Sa-Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(True)
+        elif re.search(r'^([0-9]{4})-?(1[0-2]|0[1-9])-?(3[01]|0[1-9]|[12][0-9])$', service):
+            service = service.replace('-', '')
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetDateHasService(service)
+        else:
+            raise KeyError("Unknown service keyword: " + service)
+
+        gtfs_service.SetStartDate(self.config['feed_info']['start_date'])
+        gtfs_service.SetEndDate(self.config['feed_info']['end_date'])
+        feed.AddServicePeriodObject(gtfs_service)
+        return feed.GetServicePeriod(service)
+
+    def _load_itinerary_schedule(self, schedule, itinerary, service):
+        """
+        Load the part of the provided schedule that fits to a particular
+        itinerary.
+
+        :return times: List of strings
+        """
+        times = None
+        for trip in schedule['lines'][itinerary.route_id]:
+            trip_services = trip["services"]
+            if (trip[
+                    "from"] == itinerary.fr and trip[
+                        "to"] == itinerary.to and service in trip_services):
+                times = trip["times"]
+
+        if times is None:
+            print("Warning: Couldn't load times from schedule for route")
```

Comment:
print the `route_id` and the `service` string for debugging

Reply:
The `route_id` is printed already in the main function `add_trips_to_feed` and on the screen the information is present together.

Reply:
OK, perfect ! :)